libsweep/CMakeLists.txt

```diff
@@ -0,0 +1,19 @@
+project(sweep)
+
+cmake_minimum_required(VERSION 3.5)
```

Comment:
Most likely you need a completely different cmake file for your project (e.g. it doesn't have an install target and I haven't even tested if this works correctly with the examples). It was just a simple way for me to use Visual Studio - I probably shouldn't have checked it in in the first place.